docs/contivctl.md

```diff
@@ -0,0 +1,227 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+*Note:*
+`contivctl` Command can be found under `$GOPATH/bin` directory (`/opt/gopath/bin` in vagrant VMs) after a build. 
+Currently, `contivctl` needs to be run on the same host where `netmaster` is running
+
+## Usage
+
```

Comment:
@mapuri - agree about renaming
docs/ConfigDetails.md is obsolete I think, we shouldn't encourage using that interface. Removing that and/or replacing a reference to contivctl would be helpful in main README.md